src/configs/settings.ts

```diff
@@ -92,6 +92,451 @@ export const locationHierarchy: LocationItem[] = [
+    },
+    goal: {
+      description:
+        'Register all families & family members in all residential structures enumerated (100%) within the operational area',
+      id: 'RACD_register_families',
+      priority: 'medium-priority',
+      target: [
+        {
+          detail: {
+            detailQuantity: {
+              comparator: '>=',
+              unit: 'Percent',
+              value: 100,
+            },
+          },
+          due: '',
+          measure: 'Percent of residential structures with full family registration',
+        },
+      ],
+    },
+  },
+  larvalDipping: {
+    action: {
+      code: 'Larval Dipping',
+      description: 'Perform a minimum of three larval dipping activities in the operational area',
+      goalId: 'Larval_Dipping',
+      identifier: '',
+      prefix: 5,
+      reason: 'Investigation',
+      subjectCodableConcept: {
+        text: 'Breeding_Site',
+      },
+      taskTemplate: 'Larval_Dipping',
+      timingPeriod: {
+        end: '',
+        start: '',
+      },
+      title: 'Larval Dipping',
+    },
+    goal: {
+      description: 'Perform a minimum of three larval dipping activities in the operational area',
+      id: 'Larval_Dipping',
+      priority: 'medium-priority',
+      target: [
+        {
+          detail: {
+            detailQuantity: {
+              comparator: '>=',
+              unit: 'activit(y|ies)',
+              value: 3,
+            },
+          },
+          due: '',
+          measure: 'Number of larval dipping activities completed',
+        },
+      ],
+    },
+  },
+  mosquitoCollection: {
+    action: {
+      code: 'Mosquito Collection',
+      description:
+        'Set a minimum of three mosquito collection traps and complete the mosquito collection process',
+      goalId: 'Mosquito_Collection',
+      identifier: '',
+      prefix: 6,
+      reason: 'Investigation',
+      subjectCodableConcept: {
+        text: 'Mosquito_Collection_Point',
+      },
+      taskTemplate: 'Mosquito_Collection_Point',
+      timingPeriod: {
+        end: '',
+        start: '',
+      },
+      title: 'Mosquito Collection',
+    },
+    goal: {
+      description:
+        'Set a minimum of three mosquito collection traps and complete the mosquito collection process',
+      id: 'Mosquito_Collection',
+      priority: 'medium-priority',
+      target: [
+        {
+          detail: {
+            detailQuantity: {
+              comparator: '>=',
+              unit: 'activit(y|ies)',
+              value: 3,
+            },
+          },
+          due: '',
+          measure: 'Number of mosquito collection activities completed',
+        },
+      ],
+    },
+  },
+};
+
+/** UseContext - interface for PlanPayload.useContext[] items */
+export interface UseContext {
+  code: UseContextCodesType;
+  valueCodableConcept: string;
+}
+
+/** interface that describes plan definition objects from OpenSRP */
+export interface PlanDefinition {
```

Comment:
👍 Looks like we defined similar interfaces in different places, I may cherry pick this for the IRS branch or just update it later.

Reply:
Yeah, I dont know how best we'll manage the code before we fully integrate both branches together.  Ideas?

Reply:
I was thinking of making sure both worked together and then [lightly] refactoring IRS code as needed to use shared codes you put together for FI.